zerver/views/auth.py

```diff
@@ -157,6 +157,7 @@ def maybe_send_to_registration(
     request: HttpRequest,
     email: str,
     full_name: str = "",
+    role: int | None = None,
```

Comment:
Should we move this function to having all the parameters after email be required to come as kwargs? I'm always a bit stressed about adding something in a random position in the middle without that feature.